charts/zeebe-benchmark/values.yaml

```diff
@@ -89,6 +89,9 @@ zeebe:
     zeebe.broker.executionMetricsExporterEnabled: "true"
     zeebe.broker.data.diskUsageCommandWatermark: "0.8"
     zeebe.broker.data.diskUsageReplicationWatermark: "0.9"
+    # Configure index suffix with hour pattern, so we create every hour a new index
+    # such that ILM can clean it up quickly
+    zeebe.broker.exporters.elasticsearch.args.index.indexSuffixDatePattern: "yyyy-MM-dd_HH"
```

Comment:
Here I was not sure whether we still want to use it. For mixed benchmarks it seems to cause to create several indexes, since we want to keep the indices for one day for operate.

Reply:
https://grafana.dev.zeebe.io/d/elasticsearch/elasticsearch?orgId=1&var-datasource=prometheus&var-cluster=All&var-namespace=ck-helm-mixed&var-index=All

Reply:
The question is how well Operate copes with this new pattern itself :thinking: They read from these indices right?

Reply:
![2024-01-17_14-58_1](https://github.com/zeebe-io/benchmark-helm/assets/2758593/6ea1ab0f-1b2d-4594-af21-a88d2086d7e8)


Seems to work :)